rfcs/convergence/authoring-stories.md

```diff
@@ -261,17 +261,112 @@ export const parameters = { controls: { expanded: true } };
+an example of the proposed file tree:
+
+```
+packages/react-menu/
+├─ stories/
+│  ├─ Menu/
+│    ├─ Default.stories.tsx
+│    ├─ Controlled.stories.tsx
+│  ├─ MenuList/
+│    ├─ Default.stories.tsx
+│    ├─ WithIcons.stories.tsx
+│  ├─ tmp-icons.tsx
+├─ src/
+│  ├─ index.js
+├─ .gitignore
+├─ package.json
+├─ README.md
+├─ Spec.md
+```
+
+The name of each `stories` file should be the name of the story in storybook, for easy searching in the IDE. Since we follow
+`Component Story Format (CSF)` a different folder should be used for each component in storybook.
+
+We should invest in creating lint rules and automation that will enforce this file structure and guarantee imports
+from storybook related code do not leak into the package.
 
 ### 7. how structure stories in the storybook nav tree
 
-TBA
+> This section assumes that the storybook stories forms the core documentation of Fluent converged.
+> This may change in the future so the measures proposed in this section might no longer be relevant
+> if this assumption is no longer true.
+
+Previous proposals regarding controls state that each component will contain one Default/Playground story which should
+be the first story that is displayed when the component is selected in storybook. By default storybook's CSF will display
+the first story loaded by import when a component is selected.
+
+Following the proposal by the previous section to extract each story into its own file, the order that storybook loads
+stories is no longer deterministic. Therefore, we need to modify the root storybook configuration to use a custom
+sorting algorithm so that a story called `Default` will always be sorted to be the first story under a component.
+
+The `Default` story will be reserved and unique for each component. Each package can choose to document stories for each
+component that is exported, as stated in the previous section.
+
+#### Internal stories
+
+As mentioned in for E2E testing, we should ensure maximum coverage for all publicly viewable stories by our consumers.
+For more complex scenarios that need to be tested we should make sure that stories exist for E2E tests but should not be
+easily accessible publicly.
+
+Storybook has proposed a feature for this in [storybookjs/storybook](https://github.com/storybookjs/storybook/issues/9209)
+which will configure stories to exist in deeplink URL format, but do not appear in the nav tree. As stated in the issue,
+we can workaround before the release of this feature by modifying `manager-head.html` and set `display:none`for all
+stories with a specific DOM `id` attribute. Storybook uses the `id` attribute for each link in the nav tree, and sets
+the value to the story id.
+
+We propose to use an extra filename extension and naming convention for internal stories:
+
+```ts
+// MenuTabstops.internal.stories.tsx
+
+export const MenuTabstopsInternal = () => {
+  // story
+};
```

Comment:
just for clarification, can we mention (add to the example if needed) that those file will still need to follow storybooks `CSF`, thus containing default export?